data/json/recipes/practice/ranged.json

```diff
@@ -95,6 +113,27 @@
     "components": [ [ [ "arrow_wood_heavy", 20 ] ] ],
     "byproduct_group": [ { "item": "arrow_wood_heavy", "charges": [ 0, 4 ] } ]
   },
+  {
+    "id": "prac_crossbow_int",
+    "type": "practice",
+    "//": "Ideally, this would accept multiple arrow types and return a portion of the arrows used",
+    "activity_level": "MODERATE_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_COMBAT",
+    "name": "crossbows (intermediate)",
+    "description": "Practice the best ways of firing fletched bolts into a target.  Requires a decent quality crossbow.",
+    "skill_used": "rifle",
+    "time": "30 m",
+    "practice_data": { "min_difficulty": 2, "max_difficulty": 4, "skill_limit": 5 },
+    "autolearn": [ [ "rifle", 3 ] ],
+    "book_learn": [ [ "recipe_bows", 2 ], [ "manual_archery", 2 ], [ "book_archery", 2 ] ],
+    "tools": [
+      [ "bullet_crossbow", "crossbow", "compositecrossbow", "compcrossbow", "huge_crossbow" ],
```

Comment:
Make that a crossbow requirement

Reply:
Same as before, I've changed it to a requirement entry.